src/Analysis/Codelyzer.Analysis.CSharp/CSharpRoslynProcessor.cs

```diff
@@ -398,34 +242,63 @@ private void HandleReferences(Reference reference)
                 RootNode.References.Add(rootReference);
             }
         }
-
-        private List<UstNode> HandleGenericMembers(in SyntaxList<SyntaxNode> nodeMembers)
+        private List<UstNode> HandleGenericMembers(List<SyntaxNode> children)
         {
-            List<UstNode> members = new List<UstNode>();
+            List<UstNode> childUstNodes = new List<UstNode>();
 
-            foreach (var nodeMember in nodeMembers)
+            foreach (var child in children)
             {
-                var member = HandleGenericVisit(nodeMember);
-                if (null != member)
+                var childUstNode = HandleGenericVisit(child);
+                if (null != childUstNode)
```

Comment:
nit: standard convention for null checks has `null` as the right operand, i.e. `childUstNode != null`